### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hopalonghopper**
Added steps under OSX Notes to complete a successful build

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,8 +37,14 @@ Once the shared library has been build, you will need to place it somewhere that
 * If you want to force a 32-bit build (I'm looking at you Excel 2016 on Mac), you can do:
 
     ``cmake .. -DREFPROP_32BIT=ON``
+    
+    --Note forcing a 32-bit build is no longer required for Office 365--
+
+* You have to use numpy <= 1.15.4 as 1.16+ will not build the .dylib file
 
-* On OSX, it seems you need to use the ``homebrew`` version of ``gcc`` and ``gfortran``.  You can obtain homebrew versions of gcc and gfortran with ``brew install gcc`` once homebrew is installed
+* You will also need to provide a copy of 'libquadmath.a' in your REFPROP-cmake folder. You can do this by installing gcc9 via MacPorts, navigating to the library folder for gcc9 (/opt/local/lib/gcc9), and copying the file to REFPROP-cmake.
```
Comment (ianhbell) [inline README.md]: This is not needed, if you use the homebrew gfortran, this is handled directly; you should definitely be using homebrew.

File: README.md [hunk 2]
```diff
@@ -37,8 +37,14 @@ Once the shared library has been build, you will need to place it somewhere that
...
+
+* The 'FORTRAN' folder copied to REFPROP-cmake has to be in all caps
```
Comment (ianhbell) [inline README.md]: Not really, please see the info about the REFPROP_FORTRAN_PATH flag.  That's the recommended solution.

File: README.md [hunk 3]
```diff
@@ -37,8 +37,14 @@ Once the shared library has been build, you will need to place it somewhere that
 * If you want to force a 32-bit build (I'm looking at you Excel 2016 on Mac), you can do:
 
     ``cmake .. -DREFPROP_32BIT=ON``
+    
+    --Note forcing a 32-bit build is no longer required for Office 365--
+
+* You have to use numpy <= 1.15.4 as 1.16+ will not build the .dylib file
```
Comment (ianhbell) [inline README.md]: Really?  Can you please open an issue with the output for a build with homebrew gfortran and show the shell output?

**Comment by ianhbell** [comment]
Please also see https://github.com/usnistgov/REFPROP-cmake/blob/master/CMakeLists.txt#L156-L170

File: README.md (hunk 1 above)
Comment (hopalonghopper) [reply README.md]: Well I've gone back and forth on homebrew vs MacPorts; I never liked homebrew's old statement of "don't worry, you can trust us" when giving it sudo privileges.

File: README.md (hunk 3 above)
Comment (hopalonghopper) [reply README.md]: done. do you want me to close this pull request or leave it open?

Comment (ianhbell) [reply README.md]: Let's close this one I think...

**Pull request closed without merging**